### ISSUE TITLE
Module build failed: SyntaxError: Unexpected token

### ISSUE THREAD

**Issue by adam806** [alvarotrigo/vue-fullpage.js#8]
when i import fullpage in my project and run 'npm run dev',there is an error in the below,is there something i missed? thanks a lot
here is my code and error:

```
<template>
  <div id="app">
     <full-page :options="options">
     </full-page>
  </div>
</template>
```

```
<script>
import FullPage from '../assets/js/FullPage.vue';
components: {
    FullPage
  }
</script>
```

ERROR in ./node_modules/babel-loader/lib!./node_modules/vue-loader/lib/selector.js?type=script&index=0!./src/assets/js/FullPage.vue
Module build failed: SyntaxError: Unexpected token (34:12)

  32 |         events: constants.EVENTS.reduce((eventsHandlers, event) => {
  33 |           return {
> 34 |             ...eventsHandlers,
     |             ^
  35 |
  36 |             [event]: (...args) => {
  37 |               this.emitEvent(event, args);

 @ ./src/assets/js/FullPage.vue 3:0-111 4:0-124
 @ ./node_modules/babel-loader/lib!./node_modules/vue-loader/lib/selector.js?type=script&index=0!./src/components/app.vue
 @ ./src/components/app.vue
 @ ./src/index.js
 @ multi (webpack)-dev-server/client?http://0.0.0.0:8000 webpack/hot/dev-server ./src/index.js
Child html-webpack-plugin for "index.html":

**Comment by ebisbe** [comment]
@adam806 I had this same error when trying to import the mixins. But my import is this:
```
import FullPage from 'vue-fullpage.js'
```
You can try if it helps.

**Comment by adam806** [comment]
@ebisbe thanks,i'll try it

**Comment by yuxiaoxia** [comment]
why $

**Comment by adam806** [comment]
你从哪看出来我用jquery了 @yuxiaoxia

**Comment by alvarotrigo** [comment]
> why $?
> why use jquery?

Because fullPage.js is a jQuery plugin. It would be pretty weird if it didn't required jQuery lol.
You will have to wait until like a month or so until it gets rid of jQuery.

**Comment by rainydayDY** [comment]
I try it ,it's incompatible ie browser，and throw error like 'app.js:ReferenceError: $ is not defined'，can you tell me why? and during installation, if I want to import it correctly, I must install babel-preset-es2015, and alter .babelrc.

**Comment by alvarotrigo** [comment]
@rainydayDY it works in IE Edge. The others fail due to not being compatible with some ES 6 features.

> and throw error like 'app.js:ReferenceError: $ is not defined'

Not on [the demo page](https://alvarotrigo.com/vue-fullpage/). Sounds like you are forgetting to include jQuery.

**Comment by ekoeditaa** [comment]
Running `npm install` in the vue-fullpage.js directory solves the problem for me.

**Comment by alvarotrigo** [comment]
This should be fixed in the latest version 0.0.2
